chapters/annotations.tex

```diff
@@ -1748,37 +1748,39 @@ \subsubsection{Conversion rules}\label{conversion-rules}
    "Modelica.Electrical.Machines.BasicMachines.QuasiStationaryDCMachines",
    "Modelica.Electrical.Machines.BasicMachines.QuasiStaticDCMachines");
 convertElement("Modelica.Electrical.Machines.Interfaces.PartialBasicDCMachine",
-                "quasiStationary", "quasiStatic");
-convertElement("Modelica.Electrical.Machines.BasicMachines.QuasiStationaryDCMachines.DC_ElectricalExcited",
-                "quasiStationary", "quasiStatic");
+               "quasiStationary", "quasiStatic");
+convertElement("Modelica.Electrical.Machines.BasicMachines." +
+                "QuasiStationaryDCMachines.DC_ElectricalExcited",
```

Comment:
Warning: This might not even be valid, as constant evaluation of string concatenation isn't clearly defined for annotations.

Reply:
Ok, I agree that handle it in that issue